### PR TITLE
build: upgrade struts from 1.3.8 to 1.3.10 and fix related NPE issue

### PR DESCRIPTION
**Pull request by gongjian1981**
Upgraded Struts version from 1.3.8 to 1.3.10.
Fix a NullPointerException issue that occurred when upload client image due to the version upgrade.:wq

## Summary by Sourcery

Upgrade Struts framework to version 1.3.10 and resolve a NullPointerException issue during client image uploads by modifying form data handling.

Bug Fixes:
- Fix a NullPointerException issue related to client image uploads by adjusting the way form data is accessed.

Build:
- Upgrade Struts dependencies from version 1.3.8 to 1.3.10 in the project configuration.

### CODE REVIEW

File: src/main/webapp/casemgmt/rightColumn.jsp [hunk 1]
```diff
@@ -69,8 +69,7 @@
             <img style="cursor: pointer;" id="ci"
                  src="<c:out value="${ctx}"/><%=ClientImage.imagePresentPlaceholderUrl%>" alt="id_photo" height="100"
                  title="Click to upload new photo."
-                 OnMouseOver="document.getElementById('ci').src='../imageRenderingServlet?source=local_client&clientId=
-                     <c:out value="${clientId}"/>'"
+                 OnMouseOver="document.getElementById('ci').src='../imageRenderingServlet?source=local_client&clientId=<c:out value="${clientId}"/>'"
```
Comment (yingbull) [inline src/main/webapp/casemgmt/rightColumn.jsp:72]: won't bounce it but watch long line formatting.

File: pom.xml [hunk 2]
```diff
@@ -27,6 +27,7 @@
         <log4j2.version>2.17.0</log4j2.version>
         <slf4j.version>1.7.32</slf4j.version>
         <hapifhir.version>5.4.0</hapifhir.version>
+        <struts.version>1.3.11</struts.version>
```
Comment (yingbull) [inline pom.xml:30]: I like this setup. 👍

Comment (yingbull) [inline pom.xml:30]: I like this setup. 👍

**Review by yingbull — APPROVED** [review]

**Comment by yingbull** [comment]
guess I didn't approve fast enough - shows merge conflicts.  Can you fix, @gongjian1981 ?

**Comment by gongjian1981** [comment]
> guess I didn't approve fast enough - shows merge conflicts. Can you fix, @gongjian1981 ?

I checked and found that the conflict occurred because I deleted a class file that is no longer needed, while Seb added a '//CHECKSTYLE' comment.  I've resolved the conflict.

**Pull request merged**